org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -0,0 +1,202 @@
+package org.code.theater;
+
+public enum Instrument {
```

Comment:
This is also defined in SoundLoader.java?

Reply:
Oops